### ISSUE TITLE
link in README does not work

### ISSUE THREAD

**Issue by fucusy** [bmabey/pyLDAvis#103]
[Visualizing Topic Models](https://www.youtube.com/watch?v=tGxW2BzC_DU&index=4&list=PLykRMO7ZuHwP5cWnbEmP_mUIVgzd5DZgH)

**Comment by bmabey** [comment]
Lame.. it looks like all of the conference videos are no longer available. 

@dbickson I know that Dato/Turi got acquired by Apple but do you have any idea where the Data Science Summit videos can be found now?

**Comment by rptaheri** [comment]
Seconded! Was looking to re-watch Ben’s demo today and it was gone! Any chance we could make it public again?

**Comment by inactivist** [comment]
I've been unable to locate this video anywhere -- can someone recommend a similar presentation/video?

**Comment by jeremydouglass** [comment]
The speaker deck for "Visualizing Topic Models" (Data Science Summit & Dato 2015) is here: https://speakerdeck.com/bmabey/visualizing-topic-models

**Comment by arnaudbouffard** [comment]
Yup, would also have liked to watch the video, not only ready speaker deck 😐

**Comment by bmabey** [comment]
Yeah, I wish it was still available too but the company that put the
conference on went away and with it the youtube videos. :(
